### ISSUE TITLE
Error in Python 3

### ISSUE THREAD

**Issue by CannonballSkippy** [RobbieClarken/ttmltosrt#2]
I get the following error when trying to convert a TTML file:

```
C:\Users\user\Desktop\New folder>ttmltosrt subtitle.ttml new.srt
Traceback (most recent call last):
  File "c:\program files\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\program files\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Program Files\Python36\Scripts\ttmltosrt.exe\__main__.py", line 9, in <module>
  File "c:\program files\python36\lib\site-packages\click\core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "c:\program files\python36\lib\site-packages\click\core.py", line 697, in main
    rv = self.invoke(ctx)
  File "c:\program files\python36\lib\site-packages\click\core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\program files\python36\lib\site-packages\click\core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "c:\program files\python36\lib\site-packages\ttmltosrt\ttmltosrt.py", line 14, in convert_file
    for srt in srt_generator(in_file.read()):
  File "c:\program files\python36\lib\site-packages\ttmltosrt\ttmltosrt.py", line 27, in srt_generator
    end = parse_time(caption['end'], tick_rate)
  File "c:\program files\python36\lib\site-packages\bs4\element.py", line 997, in __getitem__
    return self.attrs[key]
KeyError: 'end'
```

This is the file: [https://gist.github.com/anonymous/77fd1df60e6f15d6ae69701b56c09b18](https://gist.github.com/anonymous/77fd1df60e6f15d6ae69701b56c09b18)

**Comment by ml-** [comment]
It's because the <p> Elements don't have the end="[TIME]" attribute.

**Comment by RobbieClarken** [comment]
Thanks @seyoum and @ml- - I'll add support for the `dur` attribute when the `end` attribute is missing.

**Comment by RobbieClarken** [comment]
This should be fixed in v1.3.0 which I have just uploaded to PyPI. Please try updating with `pip3 install -U ttmltosrt` and let me know how it goes.

**Comment by CannonballSkippy** [comment]
Works like clockwork! Thanks buddy! 😃
